docs/secrets.md

```diff
@@ -63,78 +75,37 @@ Regardless of the type of secret provider used, the executed code uses the same
 {py:func}`~mlrun.execution.MLClientCtx.get_secret()` API to gain access to the value of the secrets passed to it, 
 as shown in the above example.
 
-## Secret providers
-As mentioned, MLRun provides the user with several secret providers. Each of those providers functions differently and 
+### Secret providers
+MLRun provides several secret providers. Each of those providers functions differently and 
 has different traits with respect to what secrets can be passed and how they're handled. It's important to understand 
```

Comment:
Each of these providers functions differently and 
have different traits with respect